pants

```diff
@@ -197,20 +201,25 @@ function determine_python_exe {
 # TODO(John Sirois): GC race loser tmp dirs leftover from bootstrap_XXX
 # functions.  Any tmp dir w/o a symlink pointing to it can go.
 
-function bootstrap_venv {
-  if [[ ! -d "${PANTS_BOOTSTRAP}/${VENV_PACKAGE}" ]]; then
+function bootstrap_virtualenv {
+  if [[ ! -d "${PANTS_BOOTSTRAP}/${VIRTUALENV_ZIPAPP}" ]]; then
     (
+      green "Downloading the virtualenv zipapp"
       mkdir -p "${PANTS_BOOTSTRAP}"
       local staging_dir
       staging_dir=$(tempdir "${PANTS_BOOTSTRAP}")
       cd "${staging_dir}"
-      curl -LO "https://pypi.io/packages/source/v/virtualenv/${VENV_TARBALL}"
-      tar -xzf "${VENV_TARBALL}"
-      ln -s "${staging_dir}/${VENV_PACKAGE}" "${staging_dir}/latest"
-      mv "${staging_dir}/latest" "${PANTS_BOOTSTRAP}/${VENV_PACKAGE}"
-    ) 1>&2
+      curl -LO "${VIRTUALENV_ZIPAPP_URL}"
+      fingerprint="$(shasum -a 256 "${VIRTUALENV_ZIPAPP}" | cut -d' ' -f1)"
```

Comment:
Hrm. Do we have this dependency already? We could just use python for this since we already depend on it.

Reply:
Yeah good point. Might as well.

Reply:
Done.